Feat/ordered raised hands list

bkrith opened the pull request:
<!--
Thank you for your pull request. Please provide a thorough description below.

Contributors guide: https://github.com/jitsi/jitsi-meet/blob/master/CONTRIBUTING.md
-->

**Overview**

The idea is to organize raised hands and help speaker to avoid confusion about the priority of them.
Basically, this solution create a list of all raised hands ordered by waiting time. In this way, speaker always knows which participant raised first and who's turn is next.
Blue indicator (for raised hands) not helps enough when speaker needs to know the priority, so added a red indicator for the first in order.
In participants panel added red indicator(for the first raised hand) and hover reveals waiting time for each one.

**Note:** This PR is part of EUHack21 - Winning over the classroom with Jitsi - [Ordered Raised Hands List Project](https://platform-euhack21.bemyapp.com/#/projects/607f14e20e05120019f18de7)

**UI**
 
Speaker Stats replaced with 2 tabs dialog.

One with the Speaker Stats

![22](https://user-images.githubusercontent.com/5513284/116820656-d86a3480-ab7e-11eb-888c-6b25a3579ae7.png)

and one with the raised hands list.

![33](https://user-images.githubusercontent.com/5513284/116820665-dc965200-ab7e-11eb-91f4-977f2d9046c7.png)

Red indicator in thumbnails/tiles for visual help.

![11](https://user-images.githubusercontent.com/5513284/116820689-e9b34100-ab7e-11eb-9da5-6faea9b38a41.png)

Red indicator and hover waiting time in participants panel.

![44](https://user-images.githubusercontent.com/5513284/116820696-efa92200-ab7e-11eb-9fb5-41890fc28b16.png)

**Technically**

Added a raisedHandAt property in participant object, which update with raiseHand property. With this time value(number), able to track waiting time and order raised hand for each participant.

In react/features/speaker-stats feature:

- SpeakerStats dialog replaced with a div component(tab role).
- Added a second div component, RaiseHandStats(tab role).
- Added a DialogWithTabs component, which contains the two above(tabs/divs).

rteijeiro commented:
@idevit should we send the PR to develop branch first and then merge everything in master when it's ready?

bkrith commented:
@rteijeiro I think just need to do a PR from our feature branch directly to their master.

damencho committed:
feat: Whitelists enableUnifiedOnChrome.

Whitelists it, so we can run tests by setting it to false.

hristoterezov committed:
chore(deps) lib-jitsi-meet@latest

* fix(replaceTrack):  Don't wrap Error in Error.

https://github.com/jitsi/lib-jitsi-meet/compare/64cdf69ffba44147bc791bc7b93fe811fd4f78a2...fbf85bdcec64185431cd6012060f4d4e922c573f

horymury committed:
feat(toolbar-buttons): Add event for notify overwritten toolbar buttons

horymury committed:
fix(context-menus) Fix participant context menus/toolbar overflow menu

- on ipads, long touch open dialog now opens the context menu to the left of the thumbnail as expected
- on ipads, now we close context menus on tap out
- fix case when participant context menu's height > tileview videos' height causing scroll on videos pane
- keep toolbox open while the overflow menu is shown
- keep remote participant video thumbnail in filmstrip visible even if toolbox is hidden, if context menu is opened
- Fix bug where toolbox could be completely disabled

robertpin committed:
Show reactions buttons at all times

Don't send reactions via the channel if there's only one participant in the meeting

tmoldovan8x8 committed:
feat(android) expose channels ids

tmoldovan8x8 committed:
fix(rn) disable pip while authorising dropbox

saghul committed:
fix(shared-video) make placeholder translatable

Also fix capitalization.

saghul committed:
fix(rn,shared-video) validate URLs to avoid crashes on the native side

saghul committed:
fix(rn,conference) count fake participants when checking if lonely meeting

A shared video participant counts, so the lonely meeting invite options can
disappear.

bkrith committed:
Rebased with latest master

bkrith committed:
fix conflict

saghul committed:
feat(rn,conference) show invite button on the top navbar

paweldomas committed:
fix(conference.js): add tracks to the conference

robertpin committed:
fix(av-moderation) Advanced moderation improvements (#9935)

* Update moderation in effect notifications

Only display one notification for each media type. Display notification for keyboard shortcuts as well

* Update muted remotely notification

Display name of moderator in the notification

* Fix indentation on moderation menu

* Update text for video moderation

* Added moderator label in participant pane

* Update microphone icon in participant list

For participants that speak, or are noisy, but aren't dominant speaker, the icon in the participant list will look the same as the dominant speaker icon but will not change their position in the list

* Added sound for asked to unmute notification

* Code review changes

* Code review changes

Use simple var instead of function for audio media state

* Move constants to constants file

* Moved constants from notifications to av-moderation

bkrith committed:
Merge branch 'master' into feat/ordered-raised-hands-list

bkrith committed:
update package-lock.json
